lib/services/user_service.dart

```diff
@@ -17,6 +17,31 @@ Future<QuerySnapshot> getUsers() async {
     .getDocuments();
 }
 
+Future<QuerySnapshot> getMatches(String uid, isMentee) async {
```

Comment:
One quick fix comment! Can you make a `match_service.dart` file for the functions that call the `matches` collection and `message_service.dart` for the `messages` collection? then add a `services.dart` file where you export all the 3 services so we only have to import once when we use them in other files?